src/webapp/pages/home/HomePage.tsx

```diff
@@ -139,6 +131,31 @@ export const HomePage: React.FC = React.memo(() => {
     );
 });
 
+function updateLandingNodes(nodes: LandingNode[], permissions: LandingPagePermission[], user: User): LandingNode[] {
+    return _(nodes)
+        .map(node => {
+            const pagePermission = permissions?.find(permission => permission.id === node.id);
+            if (!user) return null;
```

Comment:
`user: User`, why the check?

Reply:
this was incase the user is undefined, but i see now that it's unnecessary